FogCreek/Glitch-Community
# Clover bead: componentize ProjectAvatar

## Links
* Remix link: https://clover-bead.glitch.me

## Changes:
* componentize ProjectAvatar

## How To Test:
* visit https://clover-bead.glitch.me
* check the new project menu
* check the search autocomplete results
* check any of the popover menus that have a project search (e.g. add project to team, add project to collection)
